ADAL/src/request/ADWebAuthRequest.m

```diff
@@ -68,7 +68,7 @@ - (void)setRequestDictionary:(NSDictionary*)requestDictionary
 
 - (void)sendRequest:(ADWebResponseCallback)completionBlock
 {
-    if ([self isGetRequest])
+    if ([self isGetRequest] && _requestDictionary && [_requestDictionary allKeys].count > 0)
```

Comment:
Otherwise, it'd do the https://whatever?(null)
We could remove the second case but I wanted to make it explicit.